main/src/ca/uwaterloo/flix/language/ast/Kind.scala

```diff
@@ -51,6 +51,8 @@ sealed trait Kind {
     case (Kind.Record, Kind.Record) => true
     case (Kind.Schema, Kind.Schema) => true
 
+    // kind vars
+    case (Kind.Var(_), _) => true
```

Comment:
With this check, do you knot render kind variable checks useless? Or maybe thats the point?

Reply:
It's something we'll likely need to revisit later. Right now, kind variables are only used where the kind cannot be inferred, and that only happens where the type variable is not used anyway (picked up by redundancy phase).